main/rabbit/definitions.json

```diff
@@ -0,0 +1,277 @@
+{
+  "rabbit_version" : "3.6.10",
+  "users" : [
+	{
+	  "name" : "guest",
+	  "password_hash" : "m8PefsW4FcXeHN8M4B8OCAxHdao=",
+	  "hashing_algorithm" : "rabbit_password_hashing_md5",
+	  "tags" : "administrator"
+	}
+  ],
+  "vhosts" : [
+	{
+	  "name" : "/"
+	},
+	{
+	  "name" : "/lidskasila"
+	}
+  ],
+  "permissions" : [
+	{
+	  "user" : "guest",
+	  "vhost" : "/lidskasila",
+	  "configure" : ".*",
+	  "write" : ".*",
+	  "read" : ".*"
+	},
+	{
+	  "user" : "guest",
+	  "vhost" : "/",
+	  "configure" : ".*",
+	  "write" : ".*",
+	  "read" : ".*"
+	}
+  ],
+  "parameters" : [ ],
+  "global_parameters" : [
+	{
+	  "name" : "cluster_name",
+	  "value" : "rabbit@ef4b8b34ac2c"
+	}
+  ],
+  "policies" : [ ],
+  "queues" : [
```

Comment:
no nemusis poustet po kazdym dockeru, staci mit data rabbitu na disku a pustis to jen jednou

nemam moc rad tyhle reseni, kdy si vsichni musi pamatovat co vsechno je potreba kde nastavit, mam radsi kdyz to vsechno funguje samo. PR udelas hned ale zapomenes na to. Uz ted je problem kdyz pribude novej cron ho nastavit na serveru kdyz to neni automaticky v deploymentu a casto se na to zapomene